### DISCUSSION TITLE
Change folder icon color

### DISCUSSION THREAD

**Discussion by kaiyulee** [kyazdani42/nvim-web-devicons#253] [Q&A]
How can I change the folder icon color 
I tried

```
 override = {
  zsh = {
    icon = "",
    color = "#428850",
    cterm_color = "65",
    name = "Zsh"
  },
  folder = {
      color = "#F66B42"
  }

```

honestly, I'm not sure which section should I put the config in, `override` seems like the right one.

**Comment by alex-courtis** [comment]
Please see `:help nvim-tree-highlight`

**Reply by alex-courtis** [reply]
Apologies... I was in the wrong context.

It doesn't look like web-devicons has a folder. Which plugin are you using?

**Reply by rdhmdhl** [reply]
@alex-courtis What is the best way to customize the folder icons? I'm using nvim-tree and nvim-web-devicons

**Reply by alex-courtis** [reply]
See `:help nvim-tree-highlight` to change the nvim-tree builtin folder colours e.g.

```vim
:hi NvimTreeOpenedFolderIcon guifg=red 
```

You can optionally enable web-devicons for folders, see `:help nvim-tree.renderer.icons.web_devicons.folder`
